### DISCUSSION TITLE
Attemmpt to Select XCL-User from Pkg: in Title Bar

### DISCUSSION THREAD

**Discussion by skaisler1** [Interlisp/medley#1396] [Q&A]
When I Invoke medley, the Pkg: is set to IL and the Rdtbl: is set to Interlisp
I select Pkg: an see a list of possible packages, one of which is XCL-USER

**(Note: There is no easy way to post images here)**

If I select XCL-USER from the list, what gets set for Pkg: is "car or nobind"
So, cannot switch via Title Bar to XCL-USER.

I am using the Windows 10/11 version with CYGWIN

**Comment by pamoroso** [comment]
Try drag&drop of an image to the edit area.
